pandas/tests/io/pytables/test_timezones.py

```diff
@@ -0,0 +1,383 @@
+import datetime
+
+import numpy as np
+import pytest
+
```

Comment:
I would move the tests out of the class (as we won't be moving them again, this file is already separated nicely)